src/platforms/pumpfun/curve_manager.py

```
@@ -0,0 +1,322 @@
+        """Get the current state of a pump.fun bonding curve.
+        
+        Args:
+            pool_address: Address of the bonding curve
+            
+        Returns:
+            Dictionary containing bonding curve state data
+        """
+        try:
+            account = await self.client.get_account_info(pool_address)
+            if not account.data:
+                raise ValueError(f"No data in bonding curve account {pool_address}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)